scripts/ci/run-integration-tests

```diff
@@ -6,48 +6,59 @@
 # concept
 
 set -e
-shopt -s globstar # to allow ** in globs
 
 PROJECT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../../" && pwd )"
 RETRIES=3
 
-function executeCypress {
-  echo "=====> Executing cypress for $1"
-  local testResultPath
-  testResultPath="cypress/$(basename "$1")-result.xml"
-  npm run cypress -- -s "$1" --reporter junit --reporter-options "mochaFile=$testResultPath"
-}
-
 function runTestFile {
   echo "==> Running tests for $1"
   i="0"
   while [ $i -lt $RETRIES ]; do
     i=$((i+1))
-    echo "===> Executing run $i"
+    echo "==> Executing run $i for $1"
     local exit_code
+    local testPath
+
+    testPath="cypress/$(basename "$1")-${i}"
 
     # We don't want to let an error in the block below to end the entire script
     set +e # undo instant error flag
 
-    executeCypress "$1"
+    npm run -s cypress -- \
+      -s "$1" \
+      --config "screenshotsFolder=${testPath}/screenshots,videosFolder=${testPath}/videos" \
+      --reporter junit \
+      --reporter-options "mochaFile=${testPath}/result.xml"
+
     exit_code=$?
+    echo "==> Cypress exit code for $1: $exit_code"
 
     set -e # redo flag
-    echo "==> $i run exit code: $exit_code"
 
     if [ $exit_code -eq 0 ]; then
+      echo "==> Finished in $i runs for $1"
       return 0
     fi
   done
-
-  echo "==> Couldn't get a success in $RETRIES retries"
+  echo "==> Couldn't get a success in $RETRIES retries for $1"
   exit 1
 }
 
+function cleanup {
+  echo 'Killing http-server'
+  kill $SERVER_PID
+  echo 'Merging results'
+  junit-merge ./cypress/*/result.xml --out ./cypress/results.xml
+  echo 'Done'
+}
+
+# Starting http-server & setting up trap
+http-server -sp 4200 dist&
+SERVER_PID=$!
+trap "cleanup $SERVER_PID" EXIT
```

Comment:
Neat 👍

Reply:
nice, you approved the old version but your review wasnt dismissed when I force-pushed :D

Reply:
now its dismissed 🤔 :(